### PR TITLE
Update research.rst

### PR DESCRIPTION
**Pull request by alabaykazakh**
Add my work https://ieeexplore.ieee.org/abstract/document/10221390 to PINN papers

### CODE REVIEW

File: docs/user/research.rst [hunk 1]
```diff
@@ -118,6 +118,7 @@ Here is a list of research papers that used DeepXDE. If you would like your pape
 PINN
 ----
 
+#. V\. Medvedev, A. Erdmann, & A. Rosskopf. `Modeling of Near-and Far-Field Diffraction from EUV Absorbers Using Physics-Informed Neural Networks <https://ieeexplore.ieee.org/abstract/document/10221390>`_. *Photonics & Electromagnetics Research Symposium (PIERS)*, pp. 297-305, 2023.
```
Comment (lululxvi) [inline docs/user/research.rst:121]: Please update the format of the title and pages. See other examples

Comment (lululxvi) [inline docs/user/research.rst:121]: Add the institute to the list above.

Comment (alabaykazakh) [reply docs/user/research.rst]: Please check the latest commit.

File: docs/user/research.rst [hunk 2]
```diff
@@ -103,7 +103,8 @@ DeepXDE has been used in
   `Zienkiewicz Institute for Modelling, Data and AI <https://www.swansea.ac.uk/science-and-engineering/research/zienkiewicz-institute-for-modelling-data-ai>`_,
   `Erich Schmid Institute of Materials Science <https://www.oeaw.ac.at/esi/erich-schmid-institute>`_,
   `Athinoula A. Martinos Center for Biomedical Imaging <https://www.martinos.org>`_,
-  `Friedrich-Alexander-Universität Erlangen-Nürnberg Research Center for Mathematics of Data <https://mod.fau.eu>`_
+  `Friedrich-Alexander-Universität Erlangen-Nürnberg Research Center for Mathematics of Data <https://mod.fau.eu>`_,
+  `Fraunhofer Institute for Integrated Systems and Device Technology IISB <https://www.iisb.fraunhofer.de/>`_
```
Comment (lululxvi) [inline docs/user/research.rst:107]: Delete the last `/` in url.

File: docs/user/research.rst [hunk 3]
```diff
@@ -112,12 +113,14 @@ DeepXDE has been used in
   `General Motors <https://www.gm.com>`_,
   `RocketML <https://www.rocketml.net>`_,
   `Saudi Aramco <https://www.aramco.com>`_
+
```
Comment (lululxvi) [inline docs/user/research.rst:116]: Remove this empty line.

**Pull request merged**